src/main/java/com/amadeus/Amadeus.java

```diff
@@ -147,4 +152,25 @@ public static Configuration builder(Map<String, String> environment) {
 
     return configuration;
   }
+
+  /**
+   * Retrieves property value from grade property file.
+   *
+   * <pre>
+   * String property = Amadeus.getSystemProperties("VERSION");
+   * </pre>
+   *
+   * @param property The required property
+   * @return a property value as a String
+   */
+  public static String getSystemProperties(String property) {
+    try {
+      Properties properties = new Properties();
+      InputStream stream = new FileInputStream(Constants.GRADLE_PROPERTIES);
+      properties.load(stream);
+      return properties.get(property).toString();
+    } catch (IOException e) {
+      return "N/A";
```

Comment:
As this function could be used to get any other property (not only the version) we don't need to handle the exception at this point to keep the general nature of the function. Then every time we call the function and there is an exception, we will need to catch it and decide how to handle it depending on the property/use case. 

So you could remove this catch/return and instead add a throws exception in the function declaration:  
```
public static String getSystemProperties(String property) throws IOException {

...

}
```